cmake/registry.cmake

```diff
@@ -0,0 +1,15 @@
+# Helper macro for querying Windows registry paths
+# Queries both 32-bit and 64-bit registry views automatically
+macro(try_set_install_prefix_from_registry output_var registry_key registry_value description)
```

Comment:
Wrapper macro looks good, but does the name for it need to be this specific? Maybe it can be more general? Like `fetch_registry_value`